### ISSUE TITLE
 Broken links to PDF and VAClinics

### ISSUE THREAD

**Issue by joduinn** [department-of-veterans-affairs/vets-website#2786]
While showing vets.gov to someone I know, they stumbled onto the following broken links:

On https://www.vets.gov/healthcare/apply/, there are 3 broken links:

1) "Mail in your application. You can download, print, and fill out the 10-10EZ, then send it to this address:". Clicking on the link for 1010ez should take you to http://www.va.gov/vaforms/medical/pdf/1010EZ-fillable.pdf. However, there is no document at that location, so user is redirected to http://error.vba.va.gov/indexfw.htm. The same broken link problem also happens a few lines below, in the other link for 1010ez "Visit the nearest VA medical center or clinic to apply in person. Download, print, and fill out the 10-10EZ.".

2) "Visit the nearest VA medical center or clinic to apply in person" has broken link. Clicking on the link for VAclinic should take you to "http://www.va.gov/directory/guide/division.asp?dnum=1", but this link is broken, so user is redirected to http://error.vba.va.gov/indexfw.htm.

cc-ing @crwallace @akainic @awong-dev @koconnor719 because of discussions this week about PDF forms... and also about ongoing change-to-metalsmith work in case that is related.

**Comment by crwallace** [comment]
@joduinn -- When I am on vets.gov none of these links are broken. What network were you on when you tried to access these? I am not on the VA-network right now and they all seem to work just fine.

**Comment by joduinn** [comment]
> @joduinn -- When I am on vets.gov none of these links are broken. What network were you on when you tried to access these? I am not on the VA-network right now and they all seem to work just fine.

@crwallace I was demo-ing using my cellphone data plan and was not on any wifi. Later that morning, I was able to reproduce on my laptop-plus-hotspot, so filed the ticket. However, I've just tried it now and both links are correctly working again. Nothing changed on our side as far as I know, but thanks for investigating.

I'm closing as "unable to reproduce"! :-(
